extensions/mssql/src/main.ts

```diff
@@ -98,15 +98,15 @@ export async function activate(context: vscode.ExtensionContext) {
 
 function generateServerOptions(executablePath: string): ServerOptions {
 	let launchArgs = [];
-	launchArgs.push('--log-dir');
-	let logFileLocation = path.join(Utils.getDefaultLogLocation(), 'mssql');
-	launchArgs.push(logFileLocation);
+	launchArgs.push('--log-file');
+	let logFile = path.join(Utils.getDefaultLogLocation(), 'mssql', 'sqltools.log');
```

Comment:
We were using `--log-dir` before so that it would create a new log file for each run. I assume we still want that behavior?

Reply:
no, the idea is to start using --log-file instead so the ui is aware of what file is being used. This is needed to hook it up so tail of the log can be shown in the UI. In the current version we keep reusing the file. This is not a concern because default log level is 'Critical' that generates little to no log so until we have log rotation reusing the existing file is a good thing.

So for now if a user sets some other log level, the user is responsible for cleaning out older logs in between startups.

Once we have log rotation that I am investigating we will change this behavior.

Reply:
let's make sure this works as expected when there are multiple Azure Data Studio processes running.  I'm assuming there will name collisions.